Podfile.lock

```diff
@@ -47,7 +47,7 @@ PODS:
   - Sourcery (1.0.3)
   - StripeTerminal (1.4.0)
   - SVProgressHUD (2.2.5)
-  - UIDeviceIdentifier (1.5.0)
+  - UIDeviceIdentifier (1.6.0)
```

Comment:
Wow, this had been "outdated" for a long time 😳 That's the version with the _[Northern Hemisphere's] Fall 2020_ devices.

Would this have resulted in missing or somehow incomplete data in our reporting?